doc/descriptors.md

```diff
@@ -33,6 +33,7 @@ Output descriptors currently support:
 - Pay-to-taproot outputs (P2TR), through the `tr` function.
 - Multisig scripts, through the `multi` function.
 - Multisig scripts where the public keys are sorted lexicographically, through the `sortedmulti` function.
+- Multisig scripts inside taproot script trees, through the `multi_a` (and `sortedmulti_a`) function.
```

Comment:
Should this also be included in the [24.0 release notes](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/24.0-Release-Notes-draft#wallet)?

Something like 
> `tr()` descriptors now support multisig scripts through the `multi_a` (and `sortedmulti_a`) function.